### PR TITLE
Kelnera/remove students

### PR DESCRIPTION
**Pull request by kelnera**
These students need to be removed from the student page on the CASS
website

### CODE REVIEW

**Comment by kelnera** [comment]
@hertelc @Kennric @bielenpa Anyone see anything wrong with this?

**Comment by hertelc** [comment]
@kelnera I looked through the students you removed and they look correct.

File: content/pages/organization/cass-students.rst [hunk 1]
```diff
@@ -503,66 +253,6 @@ coffee.
...
-has programmed in C#, SQL, and JavaScript for ODOT and Oregon Sea Grant
-projects. Outside of CASS, Rob tinkers with Node.js and composes electronic
-music.
-
-.. raw:: html
-
-  <br/><br/><br/>
-
-Cody Malick
------------
```
Comment (ramereth) [inline content/pages/organization/cass-students.rst]: Cody actually moved over to the OSL so he probably should stay but be updated to reflect that change.

**Comment by hertelc** [comment]
@ramereth good point above about Cody.  Sorry, I wasn't thinking about that he moved to OSL!

**Comment by kelnera** [comment]
@ramereth @hertelc did actually mention adjusting Cody's bio, but she told me to take it off of CASS and move it to the OSL staff page. Should we keep him on the CASS student page?

**Comment by hertelc** [comment]
Sorry @kelnera , I was incorrect.  He should stay on the student page, but needs to have his info updated as an OSL student rather than SDG.

**Comment by kelnera** [comment]
@hertelc No problem! Easy fix.

**Comment by hertelc** [comment]
@Kennric I think this is ready to merge into Master?  Lets get this published live please.

**Pull request merged**